lametro/migrations/0020_auto_20250228_0726.py

```diff
@@ -0,0 +1,75 @@
+# Generated by Django 3.2.25 on 2025-02-28 15:26
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def create_calendar(apps, schema_editor):
+    calendar_model = apps.get_model("lametro", "FiscalYearCalendar")
+    calendar_model.objects.create(title="Fiscal Year 2025 (PDF)")
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("wagtaildocs", "0012_uploadeddocument"),
+        ("lametro", "0019_boardmemberdetails"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="eventnotice",
+            name="comment_conditions",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.CharField(
```

Comment:
An unmade migration from when I was adjusting the field names for clarity. I must've missed creating this 🙄